### PR TITLE
add a type specifier where one was forgor

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

Feex!
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by IProduceWidgets** [comment]
Hold on, there may be more to this...

File: Content.Client/Effects/ColorFlashEffectSystem.cs [hunk 1]
```diff
@@ -109,12 +109,13 @@ private void OnColorFlashEffect(ColorFlashEffectEvent ev)
...
 
-#if DEBUG
             if (!TryComp(ent, out ColorFlashEffectComponent? comp))
             {
+#if DEBUG
                 DebugTools.Assert(!_animation.HasRunningAnimation(ent, AnimationKey));
-            }
 #endif
+                continue;
+            }
```
Comment (metalgearsloth) [inline Content.Client/Effects/ColorFlashEffectSystem.cs:118]:
```suggestion
            }
```

File: Content.Client/Effects/ColorFlashEffectSystem.cs [hunk 2]
```diff
@@ -124,7 +124,7 @@ private void OnColorFlashEffect(ColorFlashEffectEvent ev)
                 continue;
             }
 
-            comp = EnsureComp<ColorFlashEffectComponent>(ent);
+            EnsureComp<ColorFlashEffectComponent>(ent, out comp);
```
Comment (metalgearsloth) [inline Content.Client/Effects/ColorFlashEffectSystem.cs:127]: C++ change.

Comment (IProduceWidgets) [reply Content.Client/Effects/ColorFlashEffectSystem.cs:127]: wha?

**Pull request merged**